### DISCUSSION TITLE
Add ClimateEntityFeature.TURN_OFF

### DISCUSSION THREAD

**Discussion by gjohansson-ST** [home-assistant/architecture#981] [Entity Models]
## Background

This came up on the PR adding the toogle service to the climate entity that it would make sense to add a `TURN_OFF` ClimateEntityFeature.
PR: https://github.com/home-assistant/core/pull/100418
Arch discussion: #967

## Advantages

- Only support services `turn_off` and `toggle` if `ClimateEntityFeature.TURN_OFF`
- `ClimateEntityFeature.TURN_OFF` can be added automatically

## Proposal

- Add `ClimateEntityFeature.TURN_OFF` to the `ClimateEntity`

Ref PR: https://github.com/home-assistant/core/pull/101673

**Comment by frenck** [comment]
I miss the reasoning why this is needed? Could you explain the use case?

> toogle service to the climate entity that it would make sense to add a TURN_OFF

I do not understand that relationship. We have no feature flags for other HVAC modes either.

**Comment by gjohansson-ST** [comment]
Closing this as #982 is better outlined than this.
